spec/runtime/loader.spec.ts

```diff
@@ -60,12 +61,12 @@ describe('extractStack', () => {
       },
     });
 
-    expect(requiredAPIs).to.be.deep.equal({
-      'cloudtasks.googleapis.com': {
+    expect(requiredAPIs).to.be.deep.equal([
+      {
         api: 'cloudtasks.googleapis.com',
         reason: 'Needed for v1 task queue functions',
```

Comment:
nit: let's leave out "v1" so we can consolidate messages in mixed version codebases.